common/religion/religions/light_group.txt

```diff
@@ -272,7 +286,15 @@
 			doctrine = doctrine_no_head
 			doctrine = doctrine_pluralism_pluralistic
 			doctrine = doctrine_theocracy_lay_clergy
-			
+
+			#Magic
+			doctrine = doctrine_order_magic_accepted
+			doctrine = doctrine_life_magic_accepted
```

Comment:
Life magic could be ignored. Dwarf doesn't have druids.

Reply:
Done